### PR TITLE
Exclude AMP pages from Cloudinary URL removal when saving

### PR DESCRIPTION
**Pull request by dugajean**

### CODE REVIEW

File: php/class-cli.php [hunk 1]
```diff
@@ -61,7 +61,7 @@ public function __construct( $plugin ) {
 	 * Syncs assets with Cloudinary.
 	 * ## EXAMPLES
 	 *
-	 *     wp cloudinary sync
+	 *     Wp cloudinary sync
```
Comment (dugajean) [inline php/class-cli.php:64]: This change is to satisfy the linter.

File: php/class-cli.php [hunk 2]
```diff
@@ -102,7 +102,7 @@ public function sync( $args, $assoc_args ) {
 	 * Analyze assets with Cloudinary.
 	 * ## EXAMPLES
 	 *
-	 *     wp cloudinary analyze
+	 *     Wp cloudinary analyze
```
Comment (dugajean) [inline php/class-cli.php:105]: This change is to satisfy the linter.

File: php/class-cli.php (hunk 1 above)
Comment (pereirinha) [reply php/class-cli.php:64]: @dugajean 

This is a good catch, but I don't think this should be fixed this way. In systems case sensitive, like Linux, running the `Wp` will generate an error as the command isn't found, so we shouldn't suggest people use it.

A better approach is to define a new rule on `.phpcs.xml.dist`, like this:
```
	<rule ref="Generic.Commenting.DocComment.LongNotCapital">
		<exclude-pattern>php/class-cli.php</exclude-pattern>
	</rule>
```

What do you think?

**Review by pereirinha — APPROVED** [review]
@dugajean 

Looks good.
Thank you.

**Pull request merged**